ucsb_ds_capstone_projects_2021/projects/sansum/update2.md

```diff
@@ -16,114 +16,148 @@ jupyter:
-
-Surveys:
-  - Surveys (if enabled) will allow for an alternative data capture medium for the Study Team by enabling data capture direct from Subjects via a Survey URL
-  - There are four ways a Survey can be launched, and an email generated to (Study enrolled) survey participants with a URL to access it
-  - Survey Queue displays a list of ALL surveys in the Study on a single person
-  - Surveys can be set to appear in the Survey Queue based upon if the participant has completed a particular survey and/or if certain conditions are met (based upon data values)
-  - Public Surveys can be used for the general public for teaching purposes, such as Health Fairs, Company Parties, or School Lectures, where the study team could make a laptop(s) available for attendees to enter data anonymously
-
-Monitoring:
-  - Monitoring consists of 3 separate components which are:
-    - Source Data Verification – CRF(Case report form) or fields within the CRF which require verification to source documents by the field/site monitor.
-    - Medical Review – CRF or field data requires review by e.g. a medical reviewer.
-    - Data Review - CRF or field data which require review by a data manager.
-  - An Event cannot be closed out if monitoring has not been completed on any of the 'Required' CRFs that requiring monitoring
-
-E-signature process:
-  - The icon for eSignature will show up next to the form; as well as, the Event level, as long as the role for the user has been enabled
-  - When the forms have been completed, the user with the ability to eSign, will click on the icon and enter their password
-  - eSignatures can be erased and “resigned” if necessary
+We projected the biological data into three-dimensions and plotted the color-coded points to see if we could identify any trends between the HbA1c classes. There were no clear patterns in this 3D plot.
+
+
+<img src = "https://raw.githubusercontent.com/lawrencedlin/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/sansum/visuals/3-18-21%20Insulin%20Graphs%20Combined.png" width = 500px, height = 350px>
+
+The histograms above compare the distribution of HbA1c levels of participants treated with Insulin during research and participants not treated during research. The vertical red bar indicates the point at which HbA1c = 7, giving a better view to the distribution of individuals with lower or higher values than 7. Surprisingly, the individuals not treated have lower HbA1c levels, having a skewed right distribution when compared to their treated counterparts. This could be due to the treated participants having a greater need for Insulin, or could be due to the lower number of participants being treated than not being treated.
+
+<!-- #region id="YyJ3pMindCd4" -->
+<img src = "https://raw.githubusercontent.com/lawrencedlin/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/sansum/visuals/zipcode.png" width = 400px height = 300px>

```

Comment:
Same here

Markdown can embed images directly: https://jupyterbook.org/content/figures.html
For the image `visuals/someimg.png`, replace the HTML with `![some-text](visuals/someimg.png)`